packages/lpa-questionnaire-web-app/src/middleware/get-previous-page-path.js

```diff
@@ -0,0 +1,8 @@
+const { VIEW } = require('../lib/views');
```

Comment:
We already have a method for getting previous page, please refer to one of the other views